packages/charts/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -265,16 +266,41 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
         focusZonePropsInHoverCard={this.props.focusZonePropsForLegendsInHoverCard}
         overflowText={this.props.legendsOverflowText}
         {...this.props.legendProps}
+        canSelectMultipleLegends={this.props.legendProps?.canSelectMultipleLegends}
```

Comment:
>canSelectMultipleLegends={this.props.legendProps?.canSelectMultipleLegends} [](http://example.com/codeflow?start=8&length=75)

this is already handled in previous line. Remove here

Reply:
done